docs/tutorials/github.md

```diff
@@ -77,6 +74,14 @@ Our `profile.json` file contains a reference to our `catalog.json` file.  By def
 sed -i 's/NIST_SP-800-53_rev5_catalog.json/trestle:\/\/catalogs\/nist_rev5_800_53\/catalog.json/g' profiles/nist_rev5_800_53/profile.json
 ```
 
+Finally you can copy ready-made CI/CD workflows from the `TEMPLATES` directory into your workspace. These are the trestlebot actions that will run as we make changes to the repo contents.
+
+**For example Component Definition authoring in GitHub Actions**
+```
+cp TEMPLATES/github/trestlebot-create-component-definition.yml .github/workflows
```

Comment:
The _.github_ dir won't be generated after the trestlebot init and has been removed on line 53. So _.github_ needs to be created. Is that the expected?

Reply:
Yes. The `.github/workflows` directory would need to be created. Thanks for pointing that out. I could include a `mkdir` line.